manifests/adopt.pp

```diff
@@ -253,10 +253,10 @@
 
   case $_package_type {
```

Comment:
Unrelated, but I'm reading through this and I'm wondering what's the difference between the two: default and `.tar.gz` look the same.

Reply:
To be explicit, I think this whole case statement can be replaced by `$install_command = ['tar', '-zxf', $destination, '-C', $_basedir]`.

Reply:
I briefly discussed the idea with another engineer. The conclusion was that the code is probably obsolete and we agreed to apply your suggestion. Thanks.